asmcli/commands/experimental/vm.sh

```diff
@@ -0,0 +1,438 @@
+vm_subcommand() {
+  if [[ "${*}" = '' ]]; then
+    vm_usage >&2
+    exit 2
+  fi
+
+  init_vm
+  parse_subcommand_for_vm "$@"
+}
+
+init_vm() {
+  ASM_REVISIONS=""
+
+  EXPANSION_GATEWAY_NAME="istio-eastwestgateway"; readonly EXPANSION_GATEWAY_NAME
+  ASM_REVISION_LABEL_KEY="istio.io/rev"; readonly ASM_REVISION_LABEL_KEY
+
+  INSTALL_EXPANSION_GATEWAY=0
+  INSTALL_IDENTITY_PROVIDER=0
```

Comment:
These should be added to `context.sh`. We're trying to avoid any global variable that aren't readonly.

Reply:
Done